files/shunting-yard-variables.conf

```diff
@@ -4,4 +4,4 @@
 # Spring boot service conf: see http://docs.spring.io/spring-boot/docs/current/reference/html/deployment-install.html#deployment-script-customization-conf-file
 
 JAVA_OPTS="-Dlog4j.configurationFile=/opt/shunting-yard/conf/log4j.xml -Dlogging.config=/opt/shunting-yard/conf/log4j.xml"
-RUN_ARGS="--config=/opt/shunting-yard/conf/shunting-yard-config.yml"
+RUN_ARGS="--config=/opt/shunting-yard/conf/shunting-yard-config.yml --ct-config=/opt/shunting-yard/conf/ct-common-config.yml"
```

Comment:
why not add the `ct-config: /opt/shunting-yard/conf/ct-common-config.yml` in the shunting-yard yml file? Wouldn't that be more flexible if you want to change the location?

Reply:
Agree

Reply:
That file is being dynamically created by the Docker Script : see Startup.sh. So, that location is decided by us and not the end user.

Reply:
ok I see